synapse/config/cas.py

```diff
@@ -67,6 +69,7 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
             self.cas_protocol_version = None
             self.cas_displayname_attribute = None
             self.cas_required_attributes = []
+            self.cas_enable_registration = None
```

Comment:
This should be a boolean:

```suggestion
            self.cas_enable_registration = False
```